gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -76,6 +82,28 @@
   public boolean autoCheckForUpdates = true;
   public boolean updateAvailable = false;
   public long lastCheckForUpdates = 0; // milliseconds since midnight, January 1, 1970 UTC.
+  public String analyticsClientId = ""; // Empty means do not track.
+
+  /**
+   * addListener registers the listener for changes.
```

Comment:
"Registers the listener for changes." and "Removes the listener for chagnes." below. Javadoc automatically adds in the function names and expects the first sentence to be a "verb phrase" for methods and a "noun phrase" for classes. See section 7.2 https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

(and TIL, the tip on 7.2, I've been doing that lots...)

Reply:
Done